slingr-stack/veritone-package
# Migration of Veritone

Pull-request for issue https://github.com/slingr-stack/platform/issues/15546 created by @pasaperez-slingr 

## What it does?

Duplicates the endpoint functionality of veritone and veritone-per-user migrating to package the logic.

## How to test it?

Go to the `test442` `runtime` of `idea2`. And click on the `Sign in` header action. Verify in the app monitor that after logging in, the encoded credentials are shown in the storage. 
Return to the runtime and execute the `Log out package` header action. Verify that the storage record has been deleted correctly.
Sign in again.

Execute in the console the following script that should appear in the readme of the package.

```js
var response = pkg.veritone.api.post("/v3/graphql", {query:"{graphqlServiceInfo {buildInfo}}"})
log(JSON.stringify(response))
```

Remember to use this [checklist](https://docs.google.com/document/d/139rKhIsvpT3yBSB88BBZfy03fD_Sried0bVwIWVaHJk/edit?usp=sharing) for the review.

## Technical notes

There aren't technical notes

## Deployment notes

Create a new tag v2.0.0
